ggml-opencl.c

```diff
@@ -0,0 +1,221 @@
+#include "ggml-opencl.h"
+
+#define CL_TARGET_OPENCL_VERSION 110
+#include <clblast_c.h>
+
+#include <stdio.h>
+#include <string.h>
+
+#include "ggml.h"
+
+#include <ggml_clblast_dequant.cl>
```

Comment:
I still think this should be a `"..."` include and not `<...>` even if it compiling successfully.
Otherwise I have error squiggles in VS Code because the include path is not using `-isystem`